cmd/kubeadm/app/constants/constants.go

```diff
@@ -134,6 +134,10 @@ const (
 	// It's copied over to kubeadm until it's merged in core: https://github.com/kubernetes/kubernetes/pull/39112
 	LabelNodeRoleMaster = "node-role.kubernetes.io/master"
 
+	// LabelNodeRoleExcludeBalancer specifies that the node should be
+	// exclude from load balancers created by a cloud provider.
+	LabelNodeRoleExcludeBalancer = "node.role.kubernetes.io/exclude-balancer"
```

Comment:
@kubernetes/sig-network-api-reviews 

This doesn't follow the convetions of other labels in this file, nor does it have any versioning. Are we committing to this API?

Reply:
Sorry I screwed up the cut/paste I guess. I'm happy to add the dash instead of the dot...